$MirOS: ports/www/lynx/patches/patch-lynx_help_keystrokes_environments_html,v 1.1 2010/12/23 20:25:31 tg Exp $

	Fix spelling; suggested for upstream inclusion.

--- lynx_help/keystrokes/environments.html.orig	Mon Feb 13 02:45:53 2012
+++ lynx_help/keystrokes/environments.html	Sat Mar  3 18:20:22 2012
@@ -17,7 +17,7 @@
 
 <em>ENVIRONMENT</em>
        In addition to various  "standard"  environment  variables
-       such as HOME, PATH, USER, DISPLAY, TMPDIR, etc, Lynx utilizes
+       such as HOME, PATH, USER, DISPLAY, TMPDIR, etc, Lynx utilises
        several Lynx-specific environment variables, <a href=
 "#env">if they exist</a>.
 
@@ -120,7 +120,7 @@ Environment Variables Used By Lynx:
                            If set, this  variable  specifies  the
                            default  NNTP server that will be used
                            for USENET news  reading  and  posting
-                           with Lynx, via news: URL's.
+                           with Lynx, via news: URLs.
 
        ORGANIZATION
                            This  variable,  if  set, provides the
